internal/host/statemachine.go

```diff
@@ -295,12 +295,28 @@ func NewHostStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		PostTransition:   th.PostRefreshHost(statusInfoPreparingForInstallation),
 	})
 
+	// Unknown validations
```

Comment:
The `unknown` term is really confusing.

Reply:
Any better one?

Reply:
Why is it confusing?
The term is used also in k8s API, where a condition's status can be true, false, or unknown.

Reply:
I am unfamiliar with this term.
In case it's something that is supposed to be common then that's fine. But I'm pretty sure that like me there will be other contributors that won't understand that right away.